src/js/actions/ChaosActions.js

```diff
@@ -0,0 +1,89 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+var request = require("../helpers/qajaxWrapper");
+
+var config = require("../config/config");
+
+var AppsStore = require("../stores/AppsStore");
+var ChaosEvents = require("../events/ChaosEvents");
+
+function requestApp(appId, onSuccess, onError) {
+  request({
+    url: config.apiURL + "v2/apps/" + appId
+  })
+  .success(function (data) {
+    onSuccess(data.body.app);
+  })
+  .error(function () {
+    onError();
+  });
+}
+
+function deleteTasks(appId, taskIds = [], onSuccess, onError) {
```

Comment:
This would trigger several events on the pages what we don't want in this case, also AppsStore.currentApp would be overriden, what we don't want.